debian/control

```diff
@@ -0,0 +1,24 @@
+Source: libjsonv
+Priority: optional
+Maintainer: Remy Chibois <chybz@chybz.net>
+Build-Depends: debhelper (>= 9), cmake
+Standards-Version: 3.9.5
+Section: libs
+Homepage: https://github.com/tgockel/jsonv
```

Comment:
That's perfectly okay (source code was just my personal preference ;) ). I'll update this tomorrow